src/Elements/Geometry/Mesh.cs

```diff
@@ -216,6 +218,74 @@ public Mesh()
             // An empty mesh.
         }
 
+        /// <summary>
+        /// Construct a mesh from an STL file.
+        /// </summary>
+        /// <param name="stlPath">The path to the STL file.</param>
+        /// <param name="unit">The length unit used in the file.</param>
+        /// <returns></returns>
+        public static Mesh FromSTL(string stlPath, LengthUnit unit = LengthUnit.Millimeter)
+        {
+            List<Vertex> vertexCache = new List<Vertex>();
+            var mesh = new Mesh();
+
+            var conversion = 1.0;
+            switch(unit)
```

Comment:
it feels like this should live in `Units` — e.g. `Units.GetConversionFactor(LengthUnit from, LengthUnit to)` or similar. Or alternately/additionally, a `public Func<double> Units.GetConverter(LengthUnit from, LengthUnit to)` so I could just:
```
var converter = Units.GetConverter(unit, LengthUnit.Meters);
var x =  converter(double.Parse(splits[1]));
etc.
```